smart_card_connector_app/src/manifest.json.template

```diff
@@ -24,6 +24,7 @@
     "alwaysOnTopWindows",
     "loginState",
     "usb",
+    "browser",
```

Comment:
nit: Let's place it between "alwaysOnTopWindows" and "loginState", in order to keep the list sorted.